docs/src/quickstart.md

```diff
@@ -160,6 +161,63 @@ plot_energy(idata);
 gcf()
 ```
 
+### Additional information in Turing.jl
+With a few more steps, we can use Turing to compute additional useful groups to add to the [`InferenceData`].(@ref).
+
+To sample from the prior, one simply calls `sample` but with the `Prior` sampler:
+```@example quickstart
+prior = sample(param_mod, Prior(), nsamples; progress = false)
+```
+To draw from the prior and posterior predictive distributions we can instantiate a "predictive model", i.e. a Turing model but with the observations set to `missing`, and then calling `predict` on the predictive model and the previously drawn samples:
+```@example quickstart
+# Instantiate the predictive model
+param_mod_predict = turing_model(J, similar(y, Missing), σ)
+# and then sample!
+prior_predictive = predict(param_mod_predict, prior)
+posterior_predictive = predict(param_mod_predict, turing_chns)
+```
+And to extract the elementwise log-likelihoods, which is useful if you want to compute metrics such as [`loo`](@ref),
+```@example quickstart
+loglikelihoods = Turing.elementwise_loglikelihoods(param_mod, turing_chns)
+```
+This can then be included in the `from_mcmcchains` call from above:
+```@example quickstart
+# Convert into format compatible with ArviZ.jl
+using LinearAlgebra
+# Ensure the ordering of the loglikelihoods matches the ordering of `posterior_predictive`
+loglikelihoods_vals = getindex.(Ref(loglikelihoods), string.(keys(posterior_predictive)))
+# Reshape into `(num_chains, num_samples, num_params)`
+loglikelihoods_arr = permutedims(cat(loglikelihoods_vals...; dims = 3), (2, 1, 3))
+
+idata = from_mcmcchains(
+    turing_chns,
+    posterior_predictive = posterior_predictive,
+    log_likelihood = loglikelihoods_arr,
```

Comment:
I missed it earlier, but when passing an array to `InferenceData`, we need to tell it if it which variable in `observed_data` it should map to:
```suggestion
    log_likelihood = Dict("y" => loglikelihoods_arr),
```